Lake/Config/TargetConfig.lean

```diff
@@ -11,21 +11,21 @@ namespace Lake
 /-- A custom target's declarative configuration. -/
 structure TargetConfig where
   /-- The name of the target. -/
-  name : WfName
+  name : Name
   /-- The name of the target's package. -/
-  package : WfName
+  package : Name
   /-- The type of the target's build result. -/
   resultType : Type
   /-- The target's build function. -/
   target : BuildTarget resultType
   /-- Proof that target's build result is the correctly typed target.-/
   data_eq_target : CustomData (package, name) = ActiveBuildTarget resultType
 
-family_def _nil_ : CustomData (&`✝, &`✝) := ActiveOpaqueTarget
+family_def _nil_ : CustomData (`«✝», `«✝») := ActiveOpaqueTarget
```

Comment:
I forgot to mention this, but the `` &`✝ `` syntax notation for `Name.anonymous`, not a literal tombstone.

Reply:
Thanks for noticing!  This was very nonobvious.

Reply:
I had hoped the syntax would be slightly intuitive given that the tombstone is used in the infoview for anonymous names, but I guess hope was misplaced. 😅

Reply:
Are you referring to names like `a✝`?  Those are *inaccessible*, not anonymous.  (And even if the terminology was the same, they're two very different concepts.)

Reply:
That was what I was referring to, yes. Sorry about my terminology mix-up. I know the concepts are different, however, I felt there was a reasonable connection between the two in that both cannot be written down by the user as a literal. Thus, I stole the symbol. However, given this back and forth that was probably not the wisest idea. 😓